src/Content/Content.js

```diff
@@ -1,14 +1,63 @@
-import React from 'react';
-import  './content.scss';
+import './content.scss';
 
 
+const Greeting = ({ name }) => {
+  const hour = new Date().getHours();
+  let partDay = '';
+  if (hour > 22.00 && hour < 3.00) {
+    partDay = 'night';
+  } else if (hour > 3.00 && hour < 12.00) {
+    partDay = 'morning';
+  } else if (hour > 12.00 && hour < 18.00) {
+    partDay = 'afternoon';
+  } else if (hour > 18.00 && hour < 22.00) {
+    partDay = 'evening';
+  }
+  return <p>Good { partDay }, { name }!</p>;
+};
 
-const Time = () => <time>{ new Date().toTimeString() }</time>; // если не большой компонент, то можно в одну строку и подключить в этом же файле, а не создавать отдельный файл для него
+const Number = ({
+  from, to, odd, even
+}) => {
+  const num = [];
+  for (let i = from; i < to; i++) {
+    if (odd) {
+      if ((i % 2) === 0) {
+        num.push(<li>{ i }</li>);
+      }
+    } else if (even) {
+      if (!((i % 2) === 0)) {
+        num.push(<li>{ i }</li>);
+      }
+    } else {
+      num.push(<li>{ i }</li>);
+    }
+  }
+  return <ul>{ num }</ul>;
```

Comment:
empty line before return

Reply:
@iliatcymbal ESLint show arror if I add empty line

Reply:
@Alena1508 only in this place or everywhere (e.g. in Greeting you have empty line )?
I assume it happens because you have spaces on empty line (but you shouldn't)